src/core.h

```diff
@@ -112,6 +112,36 @@ void audio_sdp_attr_decode(struct audio *a);
 int aucodec_print(struct re_printf *pf, const struct aucodec *ac);
 
 
+/*
+ * Audio Receiver Pipeline
+ */
+
+struct audio_recv;
+
+int aur_alloc(struct audio_recv **aupp, const struct config_audio *cfg,
+	      size_t sampc, uint32_t ptime);
```

Comment:
for a new person it is difficult to understand what `aur` means.

perhaps a bit longer, for example `aurecv`.